src/java/org/apache/cassandra/net/Verb.java

```diff
@@ -344,6 +344,9 @@ public boolean isResponse()
 
     Verb toPre40Verb()
     {
+        assert isResponse() || id < 38 : "unsupported pre40 request verb " + id;
+        if (PING_REQ == this)
+            return ECHO_REQ;
```

Comment:
this is not the correct approach.

background: CASSANDRA-13993 → CASSANDRA-14447 

C* 3.x originally didn't have PING_* verbs, but it was back-ported.
DSE 6.x has not back-ported it.  (I'd like to avoid if possible solving this by forcing DSE 6.x upgrades to have to upgrade to the latest patch version before upgrading to HCD 1.2)